cognito-pre-token-generation/src/test/java/no/unit/nva/cognito/TriggerHandlerTest.java

```diff
@@ -314,7 +314,7 @@ private String compareAssignedRoles(List<String> rolesBeforeLogin, List<String>
 
     private UserDto createUserWithCustomRole(String manuallyAssignedRole) throws InvalidEntryInternalException {
         UserDto sampleUser = createUserWithInstitutionAndCreatorRole();
-        List<RoleDto> roles = sampleUser.getRoles();
+        Set<RoleDto> roles = sampleUser.getRoles();
```

Comment:
User roles become a set because order is not always kept in tests and order is not important